customScripts/expansionsetup.gd

```diff
@@ -201,11 +201,6 @@ func expandGame():
 				globals.backwardscompatibility.backwardsCompatibility(person)
 			globals.expansion.updatePerson(person)
 
-	#v1.5b
```

Comment:
This code is reverted.

Reply:
checked out from main and changed/added only what I wanted for next commit.

Reply:
abandoned changes for now